core/git_command.py

```diff
@@ -245,38 +254,40 @@ def _repo_path(self, throw_on_stderr=True):
         Return the absolute path to the git repo that contains the file that this
         view interacts with.  Like `file_path`, this can be overridden by setting
         the view's `git_savvy.repo_path` setting.
-        """
-        def invalid_repo():
-            if throw_on_stderr:
-                raise ValueError("Unable to determine Git repo path.")
-            return None
 
+        Do not raise error when throw_on_stderr is `False`. It is needed
+        in GsUpdateStatusBarCommand, otherwise, spurious popup will be shown.
+        """
         # The below condition will be true if run from a WindowCommand and false
         # from a TextCommand.
         view = self.window.active_view() if hasattr(self, "window") else self.view
+        window = view.window()
```

Comment:
Can be moved closer to 280 (`if window` since it doesn't seem to be used elsewhere).